saleor/giftcard/tasks.py

```diff
@@ -4,8 +4,10 @@
 from ..celeryconf import app
 from .events import gift_cards_deactivated_event
 from .models import GiftCard
+from .search import update_gift_cards_search_vector
 
 task_logger = get_task_logger(__name__)
+GIFT_CARD_BATCH_SIZE = 10
```

Comment:
Is it expected that the batch is so small? Why?

Reply:
My bad, it was for testing purpose